### DISCUSSION TITLE
noob: saving two entities labels as linked?

### DISCUSSION THREAD

**Discussion by SanAdh13** [explosion/spaCy#11599] [Help: Coding & Implementations] [answered]
new to spacy and nlp as a whole and i was wondering if this scenario is possible?

I have been working on a custom ner label 'FOOD' and combined it with en_web_core_trf. 

when an input is ` I ate two ham sandwiches and 500g of ribeye steak and an apple`
The dictionary after sorting [using](https://stackoverflow.com/questions/59068687/spacy-most-efficient-way-to-sort-entities-by-label) are:  `{'CARDINAL': [two], 'FOOD': [ham sandwiches, ribeye steak, apple ], 'QUANTITY': [500g ]}`

Is there a way that i can have this so there is some sort of linking?
like for a certain FOOD ent  it refers to its relevant cardinal or quantity

for example. If i was to write to a text file it'll be like ` "ribeye steak : 500g " , "ham sandwich : two " and "apple: 1"` . 
Apologies if this is the wrong category or a nonsensical question

**Comment by polm** [comment, marked as answer]
It sounds like you want to build a relation extraction model. We have a sample model [here](https://github.com/explosion/projects/tree/v3/tutorials/rel_component), and you can read about the problem more generally in the [Jurafsky and Martin book](https://web.stanford.edu/~jurafsky/slp3/), Chapter 17, which should give you a good overview of the field and the problem definition.
